images/node/scripts/openshift-node

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+# This launches the Kubelet by converting the node configuration into kube flags.
+
+set -euo pipefail
+
+if ! [[ -f /etc/origin/node/client-ca.crt ]]; then
```

Comment:
Could be 

```sh
if [[ ! -f /etc/origin/node/client-ca.crt && -f /etc/origin/node/bootstrap.kubeconfig ]]; then
```

if you care

Reply:
Hrm, might.  Let me come back to this, still testing this.  I need this to make the containerized tests in openshift-ansible pass in the short term, but we may remove that path totally.